RiichiBook1.tex

```diff
@@ -157,10 +166,13 @@
 
 \vspace{.2in}
 \noindent
-Cover photo \copyright ~2015 Katar\'{i}na M\'{o}zov\'{a}, Courtesy of Riichi Mahjong Slovakia. 
+Cover photo \copyright ~2015 Katar\'{i}na M\'{o}zov\'{a}, Courtesy of Riichi Mahjong Slovakia.
 
 \vspace{.2in}
+\noindent
+W~książce wykorzystano rysunki kamieni autorstwa FluffyStuff'a\footnote{\url{https://github.com/FluffyStuff/riichi-mahjong-tiles}}, udostępnionych na licencji Creative Commons Attribution 4.0 International License\footnote{\url{https://creativecommons.org/licenses/by/4.0/}}.
```

Comment:
Use English here - we will likely want to commit this upstream to the main English version of the book.